oereb_einzelschutz_denkmal/build.gradle

```
@@ -195,7 +196,16 @@ task importDataToStage(type: Ili2pgReplace, dependsOn: "validateData") {
     models = iliModelTransferstruktur
     dbschema = "stage"
     dataFile = file(Paths.get("$rootDir", xtfFileName))
-    dataset = einzelschutzDenkmalDataSet 
+    dataset = einzelschutzDenkmalDataSet
+    finalizedBy {
+        updateS3StageLink
+    }
+}
+
+task updateS3StageLink(type: SqlExecutor) {
+    description = "Ersetzt den Link der Pdfs wieder auf die Stage-Umbebung"
```

Ich glaube, das "wieder" ist überflüssig bzw. verwirrend
```suggestion
    description = "Setzt den Link der Pdfs auf die Stage-Umbebung"
```

---

Das stimmt, aber es ist auch nicht ganz selbsterklärend, warum man im Datenumbau bereits die "Live-Umgebung" einfügt.